README.md

```diff
@@ -45,4 +45,35 @@ npm run build
 npm run server
 ```
 
-6. Go to to see the component `localhost:3004/rooms/1` 
+6. Go to to see the component `localhost:3004/rooms/1`
+
+
+# calendar
+
+## API
+
+#### URL
+
+`/rooms/:id`
+
+#### GET:
+
+`/room/:id/availability` - Retrieves the current available and unavailable dates for this location ID in body of GET request
+
+              ex: {id: number}
+
+#### PATCH:
+
+`room/:id/availability` - Updates this location's available dates based on the dates chosen on the calendar, dates chosen will be sent in PATCH body
+
+              ex: {dateStart: date, dateEnd: date}
+
+#### POST:
+
+`/room/` - Adds new location passing information that is needed in POST body, ID given by database
```

Comment:
So, is the intent of this 'POST' request to add a new listing to the database? Or is it to add something different? It seems a little ambiguous right now.

Reply:
It's to add just the calendar portion that is needed but I will make it more specific. Thank you for the feedback!